Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -59,6 +59,10 @@ xcodebuild -configuration $CONFIGURATION -workspace $NATIVEDIR/GVFS.Native.Mac.x
 USERNOTIFICATIONDIR=$VFS_SRCDIR/GVFS/GVFS.Notifications/VFSForGit.Mac
 USERNOTIFICATIONPROJECT="$USERNOTIFICATIONDIR/VFSForGit.xcodeproj"
 USERNOTIFICATIONSCHEME="VFS For Git"
+updateAppVersionCmd="(cd \"$USERNOTIFICATIONDIR\" && /usr/bin/xcrun agvtool new-marketing-version \"$VERSION\")"
```

Comment:
How does a `new-marketing-version` interact with the Info.plist? Should we move to using this to set the version of everything else that is native (the kext, log daemons, etc)?

Reply:
`new-marketing-version` updates `CFBundleShortVersionString` in the `Info.plist` file. 

>>Should we move to using this to set the version of everything else that is native (the kext, log daemons, etc)?

I can take a look at how this is done on other native code and we can discuss offline.